### ISSUE TITLE
Setup.sh script fails (because of AWS helper script?)

### ISSUE THREAD

**Issue by jonnyfux** [aws-samples/aws-scalable-big-blue-button-example#105]
**Describe the bug**
The script fails with the error below. On the installation of the BBB App stack. It looks like the reson is this active issue in a AWS helper script (https://github.com/aws/aws-parallelcluster/issues/3584) with this workaround (https://github.com/aws/aws-parallelcluster/wiki/(3.0.2,-2.11.3-and-earlier)-Custom-AMI-creation-(pcluster-build-image-or-createami)-fails-for-centos7-and-ubuntu1804)

**To Reproduce**
1. Just start the script with setup.sh

**Expected behavior**
It should start. 

```
update-alternatives: using /usr/bin/g++ to provide /usr/bin/c++ (c++) in auto mode
Setting up python-secretstorage (2.3.1-2) ...
Setting up python-keyring (10.6.0-1) ...
Setting up build-essential (12.4ubuntu1) ...
Processing triggers for man-db (2.8.3-2ubuntu0.1) ...
Processing triggers for mime-support (3.60ubuntu1) ...
Processing triggers for libc-bin (2.27-3ubuntu1.4) ...
+ pip install https://s3.amazonaws.com/cloudformation-examples/aws-cfn-bootstrap-latest.tar.gz
Collecting https://s3.amazonaws.com/cloudformation-examples/aws-cfn-bootstrap-latest.tar.gz
  Downloading https://s3.amazonaws.com/cloudformation-examples/aws-cfn-bootstrap-latest.tar.gz (465kB)
Collecting pystache>=0.4.0 (from aws-cfn-bootstrap==1.4)
  Downloading https://files.pythonhosted.org/packages/3f/e7/8750ba6c6101d6aa5ceeb20c013adf2c6f3554a12c71d75654b468404bfa/pystache-0.6.0.t$
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-u01Qua/pystache/setup.py", line 183
        command = f"pandoc -f markdown-smart --write=rst --output={rst_temp_path} {md_path}"
                                                                                           ^
    SyntaxError: invalid syntax

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-u01Qua/pystache/
Cloud-init v. 21.4-0ubuntu1~18.04.1 running 'modules:final' at Sun, 12 Dec 2021 14:43:03 +0000. Up 18.98 seconds.
2021-12-12 14:43:52,406 - cc_scripts_user.py[WARNING]: Failed to run module scripts-user (scripts in /var/lib/cloud/instance/scripts)
2021-12-12 14:43:52,406 - util.py[WARNING]: Running module scripts-user (<module 'cloudinit.config.cc_scripts_user' from '/usr/lib/python$
Cloud-init v. 21.4-0ubuntu1~18.04.1 finished at Sun, 12 Dec 2021 14:43:52 +0000. Datasource DataSourceEc2Local.  Up 67.50 seconds
```
My Workaround was replacing: 

`pip install https://s3.amazonaws.com/cloudformation-examples/aws-cfn-bootstrap-latest.tar.gz`

with

`pip3 install https://s3.amazonaws.com/cloudformation-examples/aws-cfn-bootstrap-py3-2.0-5.tar.gz`

in the bbb-on-aws-bbbappscalable.template.yaml file.

**Comment by bklein1** [comment]
Hi jonnyfux,
Thank you for reporting this issue with the remediation advice. I created a fix, that I am currently testing.

**Comment by bklein1** [comment]
Fixed with release 3.0.5

**Comment by cnm-michael** [comment]
Thanks jonnyfux for posting the workaround. I have updated the file in the s3 bucket but new instances are still deployed with the old file in /var/lib/cloud/instance/scripts/part-001. Where is the file that you updated? Thanks.
